### PR TITLE
Updating Site CSS

### LINKED ISSUE #23: CSS Themes
Color scheme needs to match the [KA brand standards](https://www.kappaalphaorder.org/brand-standards/):
  - Crimson: #BE2F37
  - Old Gold: #C19709

Beyond that, a larger color scheme needs to be developed & used to replace the default color schemes from the template

### PR DESCRIPTION
**Pull request by joe-studer-18**
Setting primary to Crimson and secondary to Old Gold. Also switching the skin.css file over to a variable based declaration for these two values to make it easier to swap color schemes later & reuse colors on new classes.

Also had to forcibly add the color scheme to all html files, since that used to be handled in javascript which will be deprecated by Issue #19.

Closes #23

### CODE REVIEW

**Comment by joe-studer-18** [comment]
Bumping this one as well @jaredwhanisch @noahklein00

**Comment by jaredwhanisch** [comment]
You may need to rebase on master and also add the class that you put into the views/valuetab/ files to the new views/abouttabs/ files.

File: active/views/about.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 <html>
 <base href="../" />
 
+<base href="../" />
+
```
Comment (jaredwhanisch) [inline active/views/about.html:6]: Might want to take this out, I believe you had added it before merging master into your branch.

**Comment by joe-studer-18** [comment]
@jaredwhanisch Ok I finally quit slacking and got around to making those changes. Shooould be g2g now.

**Pull request merged**